pkg/trace/agent/obfuscate.go

```diff
@@ -176,3 +176,62 @@ func (cco *ccObfuscator) MetaHook(k, v string) (newval string) {
 	}
 	return v
 }
+
+// MetaStructHook checks the tag with the given key and val and returns the final
+// value to be assigned to this tag.
+func (cco *ccObfuscator) MetaStructHook(k string, v []byte) (newval []byte) {
+	switch k {
```

Comment:
I prefer we stick to the left, meaning that we try to reduce indentation as much as possible. We can just do:
```go
if k != "appsec" {
    // we only obfuscate inside the "appsec" key
    return
}
// ...
```
Exit early, and the rest of the code gets less indentation.

Reply:
It was my first approach, but I found a `switch case` would be more suitable if new keys get added.

Reply:
Right, but there are no other keys now. We can use a switch later on, if more keys are added.

Reply:
Good point, I updated the code!